alibi/prototypes/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+The 'alibi.prototypes' modules includes prototypes and criticism selection methods.
+"""
+
+from .protoselect import ProtoSelect
```

Comment:
TBD - perhaps export the public utility functions here as well (unless they're quite specific to `ProtoSelect`)?

Reply:
They are specific to `ProtoSelect`. One is for cross-validation an the other is for visualization. Better to discuss whether it makes sens to export them.

Reply:
just expose `visualize_prototypes`

Reply:
Addressed in [here](https://github.com/SeldonIO/alibi/pull/626#discussion_r859867164)